packages/next/server/config-shared.ts

```diff
@@ -75,17 +75,6 @@ export interface NextJsWebpackConfig {
 
 export interface ExperimentalConfig {
   disablePostcssPresetEnv?: boolean
-  removeConsole?:
```

Comment:
Should we add a warning when the old `experimental` config is used telling them it has moved similar to how we did for swcMinify here https://github.com/vercel/next.js/blob/f7eaf006c0b7efa4e11175bd1ee4bc84082c8423/packages/next/server/config.ts#L361